packages/shared/src/lib/localLanguage.ts

```diff
@@ -57,16 +48,14 @@ export const countryCodeToVerfiedLocalLanguage = (
 ): LocalLanguage | undefined => {
     const hasCountryLanguageOrDefault = localLanguages[countryCode] || dfltLocalLanguage;
     if (
-        testName === (LocalLanguageEpicTestName || LocalLanguageBannerTestName) &&
-        variantName === (LocalLanguageEpicVariant || LocalLanguageBannerVariant) &&
-        hasCountryLanguageOrDefault
+        testName === hasCountryLanguageOrDefault?.testName &&
+        variantName === hasCountryLanguageOrDefault?.variantName
```

Comment:
The `countryCodeToVerfiedLocalLanguage` could then be refactored slightly like so:

```
export const getLocalLanguageHeaderCopy = (
	testName: string,
	variantName: string,
	countryCode: string,
	defaultBannerHeader: BannerHeaderCopy,
): BannerHeaderCopy => {
	if (testName === 'LOCAL-LANGUAGE' && variantName === 'CONTROL') {
		return localLanguageBannerHeaders[countryCode];
	}

	return defaultBannerHeader;
};
```

Reply:
Ok, simpler to read, added with countryCode check.